.github/workflows/test.yml

```diff
@@ -74,6 +74,9 @@ jobs:
           MIX_ENV: test
           DB_HOST: localhost
 
+      - name: Run format check
+        run: mix format --check-formatted
```

Comment:
we had that in the `codebase` alias already, could you check the `mix.exs` file again @bterone 

https://github.com/bterone/g-searcher/blob/develop/mix.exs#L75

Reply:
Oh, didn't see that there 🙈